README.md

```diff
@@ -577,6 +578,9 @@ If you need an app for **menstrual cycle tracking** please don't use any apps li
 ## Utilities
 - [Deskreen](https://github.com/pavlobu/deskreen) - Turn any device into a secondary screen for your computer.
 
+## Developer Tools
+- [Beekeeper Studio](https://www.beekeeper-studio.io) - Open Source SQL Editor and Database Manager with a privacy commitment in their mission statement.
```

Comment:
The URL is not correct. Could you change it to https://www.beekeeperstudio.io/ please?